### PR TITLE
Deploy Youtube Api Keys reporting tool

### PR DESCRIPTION
**Pull request by benoit74**
Deploys a cronjob which will run once a month and create the report of Youtube API keys usages as a Github Issue.

What is missing:
- the secrets which are not committed:
  - POSTGRES_URI 
  - GITHUB_TOKEN: to be created under kiwix-bot?
- the final list of assignee for Github Issue (to be confirmed)

### CODE REVIEW

File: zimfarm/report/report-youtube-api-keys.cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+              - name: GITHUB_ISSUE_LABELS
+                value: task
+              - name: CREATE_ISSUE
+                value: "true"
+              volumeMounts:
+              - name: report-youtube-api-keys-config
+                mountPath: /app/maint-scripts/report_youtube_api_keys.conf.json
+                subPath: report_youtube_api_keys.conf.json
+                readOnly: true
+              command: [/app/maint-scripts/report_youtube_api_keys.py"]
```
Comment (rgaudin) [inline zimfarm/report/report-youtube-api-keys.cronjob.yaml:38]: typo!

File: zimfarm/report/report-youtube-api-keys.cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+              envFrom:
+              - secretRef:
+                  name: report-youtube-api-keys
+              env:
+              - name: PYTHONPATH
+                value: /app
+              - name: GITHUB_REPO
+                value: openzim/zim-requests
+              - name: GITHUB_ISSUE_ASSIGNEES
+                value: qqq
```
Comment (rgaudin) [inline zimfarm/report/report-youtube-api-keys.cronjob.yaml:28]: ?

File: zimfarm/report/report-youtube-api-keys.cronjob.yaml (hunk 1 above)
Comment (benoit74) [reply zimfarm/report/report-youtube-api-keys.cronjob.yaml]: Fixed

File: zimfarm/report/report-youtube-api-keys.cronjob.yaml (hunk 2 above)
Comment (benoit74) [reply zimfarm/report/report-youtube-api-keys.cronjob.yaml]: Fixed

**Review by rgaudin — APPROVED** [review]
LGTM

**Pull request merged**